pages/reference/configuring-cloud-provider.md

```diff
@@ -95,11 +97,11 @@ Follow [these steps](https://cloud.google.com/sdk/docs/authorizing#authorize\_wi
 {% /tab %}
 
 {% tab title="AWS" %}
-_No special permissions necessary._
+No special permissions necessary, but as mentioned above, providing Plural `Administrator` access will prevent issues around application specific requirements.
 {% /tab %}
 
 {% tab title="Azure" %}
-_No special permissions necessary._
+No special permissions necessary, but as mentioned above, providing Plural `Administrator` access will prevent issues around application specific requirements.
```

Comment:
I think we might be able to say admin for the resource group you're deploying plural into here

Reply:
done